includes/polling/os/hpmsm.inc.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/*
+ * LibreNMS HP MSM OS information module
+ *
+ * Copyright (c) 2016 Mike Rostermund <mike@kollegienet.dk>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+$sysdescr = snmp_get($device, 'SNMPv2-MIB::sysDescr.0', '-Ovq');
```

Comment:
Let's merge #2637 first, then we can change the above to use the already-polled description.